pkg/common/cns-lib/vsphere/virtualcenter.go

```diff
@@ -122,6 +125,9 @@ func (vc *VirtualCenter) newClient(ctx context.Context) (*govmomi.Client, error)
 			log.Errorf("failed to load CA file: %v", err)
 			return nil, err
 		}
+	} else if len(vc.Config.Thumbprint) > 0 && !vc.Config.Insecure {
+		soapClient.SetThumbprint(url.Host, vc.Config.Thumbprint)
+		log.Infof("using thumbprint %s for url %s ", vc.Config.Thumbprint, url.Host)
```

Comment:
is it a security concern to be printing thumbprints in the logs?

Reply:
let's not print Thumbprint in the log.

Reply:
the thumbprint is a public information so it shouldn't be a security concern

Reply:
Do we really have to print out thumbprint at Info level?  Although rare, there's still a small possibility for a thumbprint to be used with a wrong cert to gain access.  Is this for debugging purpose?  It's better to change this to a debug message instead.

Reply:
@xing-yang - I'm not sure I understand what you're saying ? I don't see how you can gain access If you're able to read the thumbprint in the logs as it's a public information that is part of the cert. but let's move it to debug